usmqe_tests/web/tendrl/test_login.py

```diff
@@ -0,0 +1,89 @@
+"""
+Description: Simple tests for log in on tendrl UI
+
+Author: ltrilety
+"""
+
+
+import pytest
+from selenium.webdriver.common.keys import Keys
+
+from webstr.selenium.ui.support import WaitForWebstrPage
+from usmqe.web.tendrl.loginpage import pages as loginpage
+from usmqe.web.tendrl.mainpage.landing_page.pages import get_landing_page
+
+LOGGER = pytest.get_logger('login_test', module=True)
+
+
+def test_positive_login(testcase_set, testcase_end):
```

Comment:
Why is the fixture which opens browser split into 2 testcase_set and testcase_end?